core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java

```diff
@@ -527,17 +527,46 @@ void createMeasurementEvent(
                     }
                     case NewLineProtoParser.ENTITY_TYPE_STRING:
                     case NewLineProtoParser.ENTITY_TYPE_LONG256: {
-                        Unsafe.getUnsafe().putByte(bufPos, entity.getType());
-                        bufPos += Byte.BYTES;
-                        int l = entity.getValue().length();
-                        Unsafe.getUnsafe().putInt(bufPos, l);
-                        bufPos += Integer.BYTES;
-                        long hi = bufPos + 2L * l;
-                        floatingCharSink.of(bufPos, hi);
-                        if (!Chars.utf8Decode(entity.getValue().getLo(), entity.getValue().getHi(), floatingCharSink)) {
-                            throw CairoException.instance(0).put("invalid UTF8 in value for ").put(entity.getName());
+                        final int colTypeMeta = localDetails.getColumnTypeMeta(colIndex);
+                        if (colTypeMeta == -1) {
```

Comment:
Can this -1 be a named constant so that I can find place which sets it easier?

Reply:
ok